pkg/evpn/vrf.go

```diff
@@ -26,41 +25,34 @@ import (
 	"google.golang.org/protobuf/types/known/emptypb"
 )
 
-// CreateVpc executes the creation of the VRF/VPC
-func (s *Server) CreateVpc(_ context.Context, in *pb.CreateVpcRequest) (*pb.Vpc, error) {
-	log.Printf("CreateVpc: Received from client: %v", in)
+// CreateVrf executes the creation of the VRF
+func (s *Server) CreateVrf(_ context.Context, in *pb.CreateVrfRequest) (*pb.Vrf, error) {
+	log.Printf("CreateVrf: Received from client: %v", in)
 	// check required fields
 	if err := fieldbehavior.ValidateRequiredFields(in); err != nil {
 		log.Printf("error: %v", err)
 		return nil, err
 	}
 	// see https://google.aip.dev/133#user-specified-ids
 	resourceID := resourceid.NewSystemGenerated()
-	if in.VpcId != "" {
-		err := resourceid.ValidateUserSettable(in.VpcId)
+	if in.VrfId != "" {
+		err := resourceid.ValidateUserSettable(in.VrfId)
 		if err != nil {
 			log.Printf("error: %v", err)
 			return nil, err
 		}
-		log.Printf("client provided the ID of a resource %v, ignoring the name field %v", in.VpcId, in.Vpc.Name)
-		resourceID = in.VpcId
+		log.Printf("client provided the ID of a resource %v, ignoring the name field %v", in.VrfId, in.Vrf.Name)
```

Comment:
in a creation of an object the VRF.name will be empty initially right ? So there will be no name to print.

Reply:
correct